man/prodmgr.8.rst

```diff
@@ -18,15 +18,17 @@ SYNOPSIS
 DESCRIPTION
 ===========
 
-prodmgr is the CLI for Shasta uninstall and downgrade. It works by running
-product-specific install utility container images that provide the functionality
-that can "uninstall" or "activate" a given version of a given product.
+prodmgr is the CLI for Shasta product deletion and product activation. It works by running
+a generic product deletion utility container image that provides the functionality
+that can delete a given version of a given product. For product activation, prodmgr
+launches a product specific utility container image that provides activation functionality.
 
 ARGUMENTS
 =========
 
 *ACTION*
-    The action to perform on the specified product. "uninstall" or "activate".
+    The action to perform on the specified product. "delete", "uninstall", or "activate". Note, both the "uninstall" and "activate" commands have been deprecated.
+    The "delete" and "uninstall" actions are identical.
```

Comment:
This could be reworded to make it more immediately clear that "delete" is the new "uninstall". Something like this is better IMO:

```suggestion
    The action to perform on the specified product. Supported actions are
    "delete", "uninstall" (an alias for "delete"), and "activate". The
    "uninstall" and "activate" actions are deprecated.
```

Also note that this file hard-wraps lines for readability, which I did above.